test/unity/toolkits/object_detection/test_object_detector.cxx

```diff
@@ -761,7 +761,8 @@ BOOST_AUTO_TEST_CASE(test_object_detector_predict) {
   // Create an arbitrary SFrame with test_num_examples rows, since
   // object_detector uses the number of rows to compute num_examples, which is
   // used as a normalizer.
-  gl_sframe data({{"ignored", gl_sarray::from_sequence(0, test_num_examples)}});
+  gl_sframe data(
+      {{test_image_name, gl_sarray::from_sequence(0, test_num_examples)}});
```

Comment:
could you explain?

Reply:
in the predict function we are extracting the image column (test_image_name).
But the mocked data itself doesn't have such column which causes the error ;).